src/main/java/fragments/FilterFragment.java

```diff
@@ -0,0 +1,17 @@
+package fragments;
+
+import common.BasePage;
+import org.openqa.selenium.WebDriver;
+
+public class FilterFragment extends BasePage {
+
+    public FilterFragment(WebDriver driver) {
+        super(driver);
+    }
+
+    private static final String SHOES_CATEGORY = "//div[@class='filter-options']//span[normalize-space(text())='Shoes']";
```

Comment:
Make locator universal
```suggestion
    private static final String SHOES_CATEGORY = "//div[@class='filter-options']//span[normalize-space(text())='%s']";
```